src/main/java/com/dev/cinema/dto/MovieRequestDto.java

```diff
@@ -1,6 +1,9 @@
 package com.dev.cinema.dto;
 
+import javax.validation.constraints.NotNull;
+
 public class MovieRequestDto {
+    @NotNull
```

Comment:
please add message to @NotNull - (message = "value should not be null") or whatever

Reply:
done